src/db/service/bookmark.js

```diff
@@ -0,0 +1,53 @@
+import models from '../models';
+
+const {
+  Bookmark,
+  Art,
+} = models;
+
+module.exports = {
+  async addBookmark(artId, userId) {
+    const queryResponse = await Bookmark.findOrCreate({
```

Comment:
Since most of these services have almost the same methods (findOrCreate, destroy, get, ...). Can we create a query class that these classes will inherit from?

That will be another story though.

Reply:
Yes, we can. The team decided that once most of the work is done we will begin to refactor.